Find a functionally equivalent code:android.content.res.ColorStateList.createFromXml:COMMENT
Method Modifier: public      static      
Comment:/**
 * Create a ColorStateList from an XML document, given a set of {@link Resources}.
 */

@@ -1,5 +1,5 @@
 {
-    AttributeSet attrs = Xml.asAttributeSet(parser);
+    final AttributeSet attrs = Xml.asAttributeSet(parser);
     int type;
     while ((type = parser.next()) != XmlPullParser.START_TAG && type != XmlPullParser.END_DOCUMENT) {
     }

